qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -570,90 +570,88 @@ def __init__(self, name, instrument_x, instrument_y,
         # Get and set parameters for the set points of the coordinates
         self.add_parameter(
             'cartesian',
-            get_cmd=partial(self._get_setpoints, 'x', 'y', 'z'),
-            set_cmd=self._set_cartesian,
+            get_cmd=partial(self._get_setpoints, ['x', 'y', 'z']),
```

Comment:
Mutable default arguments to functions always makes me slightly nervous. These are never modified so you could make them tuples right?

Reply:
mutable defaults? Where do you see that? I guess I can change that to a tuple without problems

Reply:
Or will partial create a function with a list as default arguments? I did not think of that... :-)

Reply:
@sohailc yes that's my thinking, It's a small issue for sure since this never modified the input arg